src/OpenTelemetry.Abstractions/Trace/ITracerFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="ITracerFactory.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Trace
+{
+    using OpenTelemetry.Context;
+    using OpenTelemetry.Context.Propagation;
+
+    /// <summary>
+    /// Creates Tracers for an instrumentation library.
+    /// </summary>
+    public interface ITracerFactory
+    {
+        /// <summary>
+        /// Returns an ITracer for a given name and version.
+        /// </summary>
+        /// <param name="name">Name of the instrumentation library.</param>
+        /// <param name="version">Version of the instrumentation library (optional).</param>
+        /// <returns>Tracer for the given name and version information.</returns>
+        ITracer GetTracer(string name, string version = null);
```

Comment:
should version be a string or may be [Version](https://docs.microsoft.com/en-us/dotnet/api/system.version?view=netframework-4.8)

Reply:
During the spec / rfc process we agreed on using the version format proposed by @austinlparker in https://github.com/open-telemetry/oteps/pull/38/files.
This includes a prefix in addition to the actual version (e.g. "semver:1.2.3" or "git:8ae73a").
Is that OK for you?